### ISSUE TITLE
A11y: Non-interactive element <label> cannot have interactive role 'menuitem'

### ISSUE THREAD

**Issue by thdoan** [jsx-eslint/eslint-plugin-jsx-a11y#933]
I'm getting this warning in VS Code for a menu item that _is_ interactive by way of `tabindex="0"` (i.e., I can tab to all three menu items using the keyboard):

```
<ul id="menu" role="menu" aria-labelledby="menu-button">
  <li role="presentation"><label for="..." tabindex="0" role="menuitem"> Menu Item 1</label></li>
  <li role="presentation"><a href="..." role="menuitem"> Menu Item 2</a></li>
  <li role="presentation"><a href="..." role="menuitem"> Menu Item 3</a></li>
</ul>
```

Should this warning be suppressed if the element has `tabindex` >= 0?

**Comment by thdoan** [comment]
Update: This fixed the warning:

```
<ul id="menu" role="menu" aria-labelledby="menu-button">
  <li role="menuitem"><label for="..." tabindex="0"> Menu Item 1</label></li>
  <li role="menuitem"><a href="..."> Menu Item 2</a></li>
  <li role="menuitem"><a href="..."> Menu Item 3</a></li>
</ul>
```
